crates/fluvio-cli-common/src/http.rs

```diff
@@ -42,6 +51,58 @@ pub async fn read_to_end(response: Response<AsyncBody>) -> std::io::Result<Vec<u
     Ok(body)
 }
 
+pub async fn wait_http_ready(url: &str, timeout: Duration) -> Result<bool> {
```

Comment:
Not show stopper, but this could be abstracted out into utility struct since this is related to URL. So something like this

```
let url_util = HttpUrlUtil::new(ur);
url_util.wait_ready()?;
```

Reply:
Done. However, I called it "HttpsReadinessCheck" instead of "HttpUrlUtil"